steps/terraform-precheck.yaml

```diff
@@ -0,0 +1,65 @@
+parameters:
+  - name: adolibrary
+    default: cnp-azuredevops-libraries
+
+  - name: environment
+    default: sandbox
+
+  - name: keyvaultName
+    default: ''
+
+  - name: keyvaultSecret
+    default: ''
+
+  - name: location
+    default: 'UK South'
+    values:
+      - 'UK South'
+      - 'UK West'
+
+  - name: serviceConnection
+    default: ''
+
+  - name: tfversion
+    default: ''
+
+steps:
+  - checkout: self
+  - checkout: ${{ parameters.adoLibrary }}
+  - bash: echo ${{ parameters.environment }}
+
+  - task: TerraformInstaller@0
```

Comment:
why are we installing terraform in pre-check stage?

Reply:
Can't see any specific reason why we currently do in existing pipelines and given there are no dependencies with other pre-check tasks, can be removed.

Reply:
Right actually, the TF lint script requires terraform client.